testsuite/features/step_definitions/salt_steps.rb

```diff
@@ -29,18 +29,6 @@
   $output, _code = $server.run("cat #{filename}")
 end
 
-When(/^I stop salt-minion on "(.*?)"$/) do |minion|
-  node = get_target(minion)
-  pkgname = $use_salt_bundle ? "venv-salt-minion" : "salt-minion"
-  os_version = node.os_version
-  os_family = node.os_family
-  if os_family =~ /^sles/ && os_version =~ /^11/
-    node.run("rc#{pkgname} stop", check_errors: false)
-  else
-    node.run("systemctl stop #{pkgname}", check_errors: false)
-  end
-end
```

Comment:
Do/could we not need this in the future?

Reply:
There's not really a point to stop salt-minion when it's done automatically.

Also, You Are not Gonna Need It (and there's git history if you do).

Reply:
Be aware, this command seems to still be in use in other features throughout the testsuite if I'm not mistaken

Reply:
Fixed. Good catch @elariekerboull and @nodeg .